package.json

```diff
@@ -29,25 +29,18 @@
     "build-mod-descriptor": "stripes mod descriptor --full --strict | jq '.[]' > module-descriptor.json "
   },
   "devDependencies": {
-    "@babel/core": "^7.0.0-0",
+    "@babel/core": "^7.0.0",
     "@babel/eslint-parser": "^7.17.0",
-    "@babel/plugin-proposal-class-properties": "^7.13.0",
-    "@babel/plugin-proposal-decorators": "^7.13.15",
-    "@babel/plugin-transform-runtime": "^7.13.15",
-    "@babel/preset-react": "^7.13.13",
-    "@folio/eslint-config-stripes": "^6.1.0",
-    "@folio/stripes": "^8.0.0",
-    "@folio/stripes-cli": "^2.4.0",
-    "@folio/stripes-components": "^11.0.0",
-    "@folio/stripes-core": "^9.0.0",
+    "@folio/eslint-config-stripes": "^7.0.0",
+    "@folio/jest-config-stripes": "^2.0.0",
+    "@folio/stripes": "^9.0.0",
+    "@folio/stripes-cli": "^3.0.0",
+    "@folio/stripes-components": "^12.0.0",
+    "@folio/stripes-core": "^10.0.0",
     "@folio/stripes-testing": "^4.2.0",
     "@formatjs/cli": "^4.2.20",
     "@testing-library/jest-dom": "^5.12.0",
```

Comment:
Can we remove `"@testing-library/jest-dom"`? It is also added in `jest-config-stripes`.

Reply:
Tried it again and it works now...removed the dep

Reply:
It took place because `jest` itself doesn't include the `toBeInTheDocument` by default. It comes from `@testing-library/jest-dom` package. You need to change the path from `@testing-library/jest-dom` to `@folio/jest-config-stripes/testing-library/jest-dom` in the `jest.setup.js` file.

Reply:
> Tried it again and it works now

Probably you already have `@testing-library/jest-dom` in `node_modules`, but if you try to install `stripes-authority-components` separately you would get the error `toBeInTheDocument is not a function`, because `@testing-library/jest-dom` is absent in `node_modules` because there is no such dependency in `package.json`.